### PR TITLE
MH3: 2 "easy" cards

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Wumpus Aberration
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt:1]: Pretty sure I saw this card in one of the other recent PRs.

File: forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+ManaCost:3 G
+Types:Creature Eldrazi Beast
+PT:6/6
+K:Devoid
+T:Mode$ SpellCast | ValidCard$ Card.Self | CheckSVar$ CastSA>Count$Adamant_1.Colorless.0.1 | Execute$ TrigChangeZone | TriggerDescription$ When you cast this spell, if {C} wasn't spent to cast it, target opponent may put a creature card from their hand onto the battlefield.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ValidTgts$ Opponent | Chooser$ Targeted | ChangeType$ Creature | ChangeNum$ 1 | Optional$ True
+K:Trample
+DeckHints:Ability$Mana.Colorless
+Oracle:Devoid (This card has no color.)\nWhen you cast this spell, if {C} wasn't spent to cast it, target opponent may put a creature card from their hand onto the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt:10]: incomplete

File: forge-gui/res/cardsfolder/upcoming/unstable_amulet.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Name:Unstable Amulet
+ManaCost:1 R
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigEnergy | TriggerDescription$ When CARDNAME enters the battlefield, you get {E}{E} (two energy counters).
+SVar:TrigEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2
+T:Mode$ SpellCast | ValidCard$ Card.wasNotCastFromYourHand | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ Whenever you cast a spell from anywhere other than your hand, CARDNAME deals 1 damage to each opponent.
+SVar:TrigDamage:DB$ DealDamage | NumDmg$ 1 | Defined$ Opponent
+A:AB$ ChangeZone | Cost$ T PayEnergy<2> | Defined$ Imprinted | Origin$ Command | Destination$ Exile | SubAbility$ DBCleanup | StackDescription$ None | SpellDescription$ Exile the top card of your library.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | ClearImprinted$ True | SubAbility$ DBExile | StackDescription$ None
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/unstable_amulet.txt:9]: we should check if None can't be default for Cleanup if it isn't already

File: forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt (hunk 1 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt:1]: It was unclaimed in the project so I did it

File: forge-gui/res/cardsfolder/upcoming/unstable_amulet.txt (hunk 3 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/unstable_amulet.txt:9]: oops, think that is copy pasta

**Review by tool4ever — APPROVED** [review]

**Pull request merged**